cassandra-commons/src/main/java/com/mesosphere/dcos/cassandra/common/offer/ClusterTaskPlacementStrategy.java

```diff
@@ -2,28 +2,25 @@
 
 import com.google.common.collect.Lists;
 import org.apache.mesos.Protos;
-import org.apache.mesos.offer.PlacementStrategy;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 
-public class ClusterTaskPlacementStrategy implements PlacementStrategy {
+public class ClusterTaskPlacementStrategy {
     private static final Logger LOGGER = LoggerFactory.getLogger(
             ClusterTaskPlacementStrategy.class);
 
-    @Override
-    public List<Protos.SlaveID> getAgentsToAvoid(Protos.TaskInfo taskInfo) {
+    public static List<String> getAgentsToAvoid(Protos.TaskInfo taskInfo) {
         return Lists.newArrayList();
     }
 
-    @Override
-    public List<Protos.SlaveID> getAgentsToColocate(Protos.TaskInfo taskInfo) {
-        List<Protos.SlaveID> agentsToColocate = new ArrayList<>();
+    public static List<String> getAgentsToColocate(Protos.TaskInfo taskInfo) {
+        List<String> agentsToColocate = new ArrayList<>();
         // Collocate this task with the corresponding Cassandra node task
-        Arrays.asList(taskInfo.getSlaveId());
+        Arrays.asList(taskInfo.getSlaveId().getValue());
```

Comment:
As someone pointed out https://github.com/mesosphere/dcos-cassandra-service/issues/229, should this line be 
`agentsToColocate(taskInfo.getSlaveId().getValue())`?